packages/config/config/devices/0x0090/918.json

```
@@ -0,0 +1,176 @@
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_auto_lock"
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_auto_lock_jammed"
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_lock_jammed"
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_all_codes_deleted"
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_code_deleted"
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_new_user_code_added"
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_new_user_code_not_added"
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_keypad_disabled"
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_new_program_code_entered"
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_manual_code_exceeds_limit"
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_messaging_user_code_entered"
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_cover_status_tampering"
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_tampering_invalid_code"
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_power_has_been_applied"
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_low_battery"
+			},
+			{
+				"$import": "templates/kwikset_template.json#alarm_map_critical_battery"
+			}
+		]
+	},
+	"metadata": {
+		"inclusion": "Initiate the process to add the lock to your system at your smart home controller. Refer to your smart home system instructions for more information. If prompted by your smart home system to add the lock, press \"button A\" on the lock interior one time. The red LED will illuminate when the lock enters Add Mode.At this time, you may be prompted by your smart home system to enter the 5-digit PIN found on the label of the interior assembly. If not, proceed to step D. If successful, re-name the lock in your system (if applicable). If unsuccessful, follow your system's instructions to remove the lock from the controller and any other network, then press \"button A\" on the lock one time",
+		"exclusion": "Removing the lock from the network\nFollow your smart home system's instructions to remove the lock from the network. When prompted by the system, press button A” on the lock interior once",
+		"reset": "Please use this procedure only when the network's primary controller is missing or otherwise inoperable.\n1. Remove Battery Pack.\n2 Press and HOLD the \"Program\" button while reinserting the battery pack. Keep holding the button for 30 seconds until the lock beeps and the status LED flashes red.\n3. Press the \"Program\" button once more. The status LED will flash green and red several times.\n4. After a few seconds, the lock will initiate the door handing process, and the latch bolt will extend and retract to learn the orientation of the door",
```

Let's shorten this a bit.
```suggestion
		"inclusion": "Press button A on the lock interior once. The red LED will illuminate when the lock enters Add Mode.",
		"exclusion": "Press button A on the lock interior once.",
		"reset": "1. Remove the battery pack.\n2. Press and hold the Program button while reinserting the battery pack. Keep holding the button for 30 seconds until the lock beeps and the status LED flashes red.\n3. Press the Program button once. The status LED will flash green and red several times.\n4. The lock will initiate the door handing process and the latch bolt will extend and retract to learn the orientation of the door.",
```

---

Happy to update! This came from an import of the Z-Wave Alliance product DB and I wasn't sure how much I should change them.

---

I've recently started to clean up all of the metadata instructions using an LLM to get them to a (mostly) consistent format. I might need to add that part as a github action and have the bot propose them to contributors.